students/todo.md

```diff
@@ -62,15 +62,19 @@ Now there's only one thing left to say: Use all the channels there are and ask a
 
 Most of the aspects mentioned here need to be done at the beginning of the program. Follow these steps and you'll have a great setup for all the activities.
 
-<h3>Github Private Repo</h3>      
-A separate Github repo is the best place for your team to organize the issues to work on. We will provide a private repo for each team so you can safely get on with your work.
+<h3>RGSoC-Orga Slack Team</h3>
+All the members of the team will be invited to our Slack RGsoC-Orga Team. This is currently our favourite channel of communication and we set up several channels with different purposes so that you won't miss any important information. You can also talk with each other via private message.
+
+We recommend your supervisor to set up a channel for the team members so that you can talk with each other on a daily basis (when we say daily, we are refering to work days). If your team doesn't like slack for internal communication, you can also use email or any other apps, however, we advise you not to lose track of RGSoC-Orga Slack channels,as most of the communication (with the organisers and other students, coaches, mentors and supervisors) will be done there.
 
 <h3>Daily Log</h3>     
 The daily log is a short list of things you do each day — a summary in the form of bullet points is perfectly fine. The Teams App will be the place to put these. 
 
 Besides keeping your team in the loop, it'll get you familiar with good team practice, and with the high level of transparency that is typical for Open Source development. It will also help you greatly with self-reflection.
 
-We recommend to either combine the daily log with [a morning standup meeting](http://martinfowler.com/articles/itsNotJustStandingUp.html) or fill the log at the end of the day. 
+We recommend to either combine the daily log with [a morning standup meeting](http://martinfowler.com/articles/itsNotJustStandingUp.html) or fill the log at the end of the day.
+
+To start making your status updates, you have to login on the RGSoC Teams app and go to Status Updates on the top of he page.
```

Comment:
We can combine this line with the intro of the daily log paragraph at the top, something like: 

"The Teams App will be the place to put these; just log into the app and go to Status Updates at the top of the page".